core/Scene.js

```diff
@@ -122,20 +158,27 @@ Scene.prototype.onReceive = function onReceive (event, payload) {
     // and the context would receive its size the same way that any render size
     // component receives its size.
     if (event === 'CONTEXT_RESIZE') {
-        
-        if (payload.length < 2) 
+        if (payload.length < 2) {
             throw new Error(
-                    'CONTEXT_RESIZE\'s payload needs to be at least a pair' +
-                    ' of pixel sizes'
+                'CONTEXT_RESIZE\'s payload needs to be at least a pair' +
+                ' of pixel sizes'
             );
+        }
+
+        this._initialized = true;
+
+        while (this._messages.length)
+            this._updater.message(this._messages.shift());
```

Comment:
This will very likely be sending irrelevant messages.

For example, if align has been set, then there are transform changes currently in the queue that will only be overwritten a few commands later.